modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/skin/Utils.java

```diff
@@ -417,12 +417,14 @@ public static String computeClippedText(Font font, String text, double width,
     }
 
     public static String computeClippedWrappedText(Font font, String text, double width,
-                                            double height, OverrunStyle truncationStyle,
+                                            double height, double lineSpacing, OverrunStyle truncationStyle,
                                             String ellipsisString, TextBoundsType boundsType) {
         if (font == null) {
             throw new IllegalArgumentException("Must specify a font");
         }
 
+        height += lineSpacing;
+
```

Comment:
This method makes a call to -
double eHeight = computeTextHeight(font, ellipsis, 0, boundsType);

What is the behavior if we make a call to the other method that takes in lineSpacing?
double eHeight = computeTextHeight(font, ellipsis, 0, lineSpacing, boundsType);

Reply:
The call only determines the ellipsis height (eHeight) to see if the space is so small that even an ellipsis wouldn't fit.  For a multi-line ellipsis (if there is such a thing) we could pass in the line spacing here.